scripts/provision-all

```diff
@@ -17,7 +17,12 @@ echo "Create base Template to be used by others"
 sudo qubesctl --show-output --targets dom0 state.sls sd-workstation-template
 
 echo "Configure packages inside base Template"
-sudo qubesctl --show-output --skip-dom0 --targets securedrop-workstation state.sls sd-workstation-template-files
+sudo qubesctl --show-output --skip-dom0 --targets securedrop-workstation-buster state.sls sd-workstation-template-files
+
+# This is required only when swapping base templates, see scripts/prep-upgrade
+# for more details.
+echo "Prep for upgrade, shut down AppVMs that require reboot for template swap"
+./scripts/prep-upgrade
```

Comment:
Recommend moving the script call _out_ of the `provision-all` script and into a Salt state, reason being that then we have the ability to trigger the change via unattended-upgrades, e.g. via dom0 state.highstate. Currently only developers run this bash script, so let's not depend on it overmuch to enforce updates we want to land on pilot/prod instances.